nimterop/cimport.nim

```diff
@@ -184,8 +228,8 @@ proc cSearchPath*(path: string): string {.compileTime.}=
   if result.len == 0:
```

Comment:
* need to distinguish whether path is absolute:
```
result = findPath(path, fail = false)
if (not path.isAbsolute) and result.len == 0: ...
```

because `"/foo" / "/some/abs/path"` is `"/foo/some/abs/path"`

Reply:
findPath already uses your joinPathIfRel that checks for absolute.